src/test/webdriver/chromedriver.rs

```diff
@@ -33,7 +33,7 @@ pub fn install_chromedriver(
 ) -> Result<PathBuf, failure::Error> {
     let target = if target::LINUX && target::x86_64 {
         "linux64"
-    } else if target::MACOS && target::x86_64 {
+    } else if target::MACOS {
```

Comment:
```suggestion
    } else if target::MACOS && target::aarch64 {
        "mac64_m1"
    } else if target::MACOS && target::x86_64 {
```

@anp The chromium issue you linked to now mentions that `chromedriver` has been released with native binaries for Apple Silicon for versions 87 and 88, and sure enough, they have: https://chromedriver.chromium.org/.

The suffixes for these builds are `mac64_m1`. `target.rs` would also have to be updated, as well as `DEFAULT_CHROMEDRIVER_VERSION`.

Reply:
Thank you for the heads up!